### DISCUSSION TITLE
What could we use to replace Rewrite with JoinFaces 5.0.x?

### DISCUSSION THREAD

**Discussion by persapiens** [joinfaces/joinfaces#1359] [General]
Rewrite support removed [here](https://github.com/joinfaces/joinfaces/commit/c9d2cb8a9f7ecfc9c05da79cf7287577f860dab9) to prepare JoinFaces to Jakarta 9.

Which library could we use now?

**Comment by larsgrefer** [comment]
Here is the corresponding discussion for rewrite itself: https://github.com/ocpsoft/rewrite/issues/304
